### ISSUE TITLE
Cannot install dvdfingerprint from pip

### ISSUE THREAD

**Issue by sjwood** [kolanos/dvdfingerprint#1]
Hi there,

I've just tried to install dvdfingerprint from pip and received the following error:

``` sh
pip install dvdfingerprint

Downloading/unpacking dvdfingerprint
  Downloading dvdfingerprint-0.1.2.tar.gz
  Running setup.py (path:/tmp/pip_build_root/dvdfingerprint/setup.py) egg_info for package dvdfingerprint
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/dvdfingerprint/setup.py", line 11, in <module>
        long_description=open('README.rst').read(),
    IOError: [Errno 2] No such file or directory: 'README.rst'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/dvdfingerprint/setup.py", line 11, in <module>

    long_description=open('README.rst').read(),

IOError: [Errno 2] No such file or directory: 'README.rst'
```

I notice from the source package on PyPI that the README.rst file appears to be missing from the gzipped tar. This seems really odd, as if I clone the dvdfingerprint repo and run the following:

``` sh
python setup.py sdist
```

then the gzipped tar includes the README.rst file. :confused:

I realise that this repo hasn't been touched for 3 years, but could you see your way to repackage and republish to PyPI please?

Thanks

**Comment by kolanos** [comment]
Hi Steve,

Thanks for bringing this to my attention. I just released 0.1.3 which
should fix it.

**Comment by sjwood** [comment]
Hi Michael,

I've just installed v0.1.3 from PyPI using pip and can confirm that this is fixed.

Thanks so much for doing this so quickly - I appreciate your attentiveness.

Regards

Steve
